### PR TITLE
Update ocenaudio from 3.7.2 to 3.7.3

### PR DESCRIPTION
**Pull request by suschizu**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

**Comment by suschizu** [comment]
`Checksum for Cask 'ocenaudio' does not match.
354Expected: 589e62c2f4785ae8dcab530dd6503fbeee6a1d70f77a4da1e5e356a9060f451a
355  Actual: d359dd72f7bd8ede0e9557b0818336f1418d66b5c50fe740916f4820857079bb`
… that's because the app for mojave

File: Casks/ocenaudio.rb [hunk 1]
```diff
@@ -1,12 +1,12 @@
 cask 'ocenaudio' do
-  version '3.7.2'
+  version '3.7.3'
 
   if MacOS.version <= :high_sierra
     sha256 '589e62c2f4785ae8dcab530dd6503fbeee6a1d70f77a4da1e5e356a9060f451a'
```
Comment (404NetworkError) [inline Casks/ocenaudio.rb]:
```suggestion
     sha256 '1bc0c0742df9a2d683e72f50bef06ca14af2ac6e5c068eb96ebe973fa52a6d37'
```
You need to update both `sha256`s for this cask when doing version changes.

File: Casks/ocenaudio.rb [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 
     url 'https://www.ocenaudio.com/downloads/index.php/ocenaudio_sierra.dmg'
   else
-    sha256 '33a08bbfb277e9f5aeae4928265d65c3d5ee47aca96fa5a6409200371f90d33b'
+    sha256 '6d0b42cb13083b7d7559cc060604a73e3e289886b64159e89d7d47b9e0fabd75'
```
Comment (404NetworkError) [inline Casks/ocenaudio.rb]:
```suggestion
    sha256 '580a44cef051ae28df615411d53f0328fb94c55a1b600894e394cde2c3862293'
```
It appears the `sha256` changed from your original commit. This is the new one I got.

File: Casks/ocenaudio.rb (hunk 1 above)
Comment (suschizu) [reply Casks/ocenaudio.rb]: thanks for your help! You are right - the <= high sierra version also got a new sha - I forgot to check this

**Pull request merged**